modules/ocf_tv/manifests/init.pp

```diff
@@ -63,4 +63,11 @@
       User['ocftv'],
     ],
   }
+
+  $pulse_sink = lookup('pulse_sink')
+
+  exec { "pactl set-default-sink ${pulse_sink}":
```

Comment:
I believe this will run every puppet run, which isn't great (although not terrible either since I assume it's fast), is there some other way this could be set, or maybe some trigger that could be made so that it only runs when needed?